backend/controllers/GameController.js

```diff
@@ -2,6 +2,21 @@ const MainController = require('./mainController/MainController');
 
 class GameController extends MainController {
 
+    generateCodeRoom = () => {
+        let codeRoom = "";
+        const possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
```

Comment:
Code room will be case-insensitive, so you can remove capital letters

Reply:
Done